src/charm.py

```diff
@@ -540,7 +555,7 @@ def _connect_logging_snap_endpoints(self):
         for plug in self._cos.snap_log_endpoints:
             try:
                 self.snap.connect("logs", service=plug.owner, slot=plug.name)
-            except snap.SnapError as e:
+            except snap_lib.SnapError as e:
```

Comment:
not sure why this rename was necessary, it sure adds a lot of noise to the PR

Reply:
Yeah this did get noisy.  It was to keep the difference between Charm.snap and library for snaps a bit clearer, but I think it mattered a lot more a few iterations ago and now not so much.  I'll change it back